modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/resources/META-INF/resources/configuration.jsp

```diff
@@ -26,10 +26,17 @@ boolean editable = PrefsParamUtil.getBoolean(PortletPreferencesFactoryUtil.getPo
 boolean formView = PrefsParamUtil.getBoolean(PortletPreferencesFactoryUtil.getPortletSetup(renderRequest), renderRequest, "formView", false);
 long formDDMTemplateId = PrefsParamUtil.getLong(PortletPreferencesFactoryUtil.getPortletSetup(renderRequest), renderRequest, "formDDMTemplateId");
 long recordSetId = PrefsParamUtil.getLong(PortletPreferencesFactoryUtil.getPortletSetup(renderRequest), renderRequest, "recordSetId");
+String scopeType = PrefsParamUtil.getString(PortletPreferencesFactoryUtil.getPortletSetup(renderRequest), renderRequest, "lfrScopeType");
 boolean spreadsheet = PrefsParamUtil.getBoolean(PortletPreferencesFactoryUtil.getPortletSetup(renderRequest), renderRequest, "spreadsheet");
 
 Group scopeGroup = themeDisplay.getScopeGroup();
 
+if (scopeType.equals("company")) {
+	scopeGroup = GroupLocalServiceUtil.getGroup(themeDisplay.getCompanyGroupId());
+
+	scopeGroupId = scopeGroup.getGroupId();
+}
+
 if (scopeGroup.isStagingGroup() && !scopeGroup.isInStagingPortlet(DDLPortletKeys.DYNAMIC_DATA_LISTS)) {
```

Comment:
@IstvanD shouldn't it be an `if/else`?

Reply:
hi @carolmariaabb I have rewritten it to `if/else`.